storage_broker/src/bin/storage_broker.rs

```diff
@@ -437,6 +438,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     info!("version: {GIT_VERSION}");
     ::metrics::set_build_info_metric(GIT_VERSION);
 
+    // On any shutdown signal, log receival and exit.
+    std::thread::spawn(move || {
+        let signals = ShutdownSignals;
+        signals.handle(|signal| {
+            info!("received {}, terminating", signal.name());
+            std::process::exit(0);
+        })
+    });
+
```

Comment:
Maybe use `install_shutdown_handlers` here? I'm not sure if it makes much difference, but that's what the pageserver and safekeeper does, so it would seem nice to be consistent.